src/lister_haxe/lister_syscmd.hx

```diff
@@ -0,0 +1,25 @@
+import Sys.*;
+
+class Lister_SysCmd {
+  static var defaultUrl : String = "http://www.google.com";
+
+  static public function main():Void {
+
+    trace('Please provide a target url, or enter for default ($defaultUrl):');
+    var input = stdin().readLine();
+    var url = switch(input) { case "": defaultUrl; case _: input; }
```

Comment:
It turns out Haxe does have a ternary operator, but I came across this behavior, `switch` returning a value, first... so this is kind of here as a demo of neato Haxe syntax.  `_` is *a lot* more powerful than just `case default:`, it's more like an F# discriminated union (I think?).  Check it out: http://haxe.org/manual/lf-pattern-matching-variable-capture.html